### DISCUSSION TITLE
API documentation

### DISCUSSION THREAD

**Discussion by corrpel** [Lillifee/raspiCam#65] [General]
Hello,
Is there any location for the API documentation?
I am trying to use http live streaming, but the endpoint http://ip:8000/api/stream/live is not working in VLC.
Also, i would like to use capture url and the delete the photo via api from the server.
Awesome app btw, thank you!

**Comment by Lillifee** [comment]
Hey, 

sorry there is no dedicated documentation page for the api...
The website supports the following commands:

/api/start start capture
/api/stop stop capture (e.g. for videos)
/api/stream/mjpeg mjpeg stream
/api/stream/live h264 stream but not viewable from the web
/api/gallery gallery
/api/gallery/delete delete items in gallery

And the different settings via get and post:

/api/camera camera settings
/api/preview previewsettings
/api/stream streamsettings
/api/vid video settings
/api/photo photo settings
/api/control control settings (mode, running etc.)

Let me know if you need more informations...

The h264 stream on /api/stream/live is not real valid format. Its just streaming the output from raspicam to the website... I'm not sure whats needed in detail to make it work.
